ui/src/wide_gauge_stat.tsx

```diff
@@ -83,8 +83,8 @@ export const
         data = unpack<Rec>(s.data),
         value = substitute(s.value, s.data),
         auxValue = substitute(s.aux_value, s.data),
-        valueNonDigitCharCount = value.match(/[^0-9]/g).length,
-        auxValueNonDigitCharCount = auxValue.match(/[^0-9]/g).length,
+        valueNonDigitCharCount = value.match(/[^0-9-]/g).length,
+        auxValueNonDigitCharCount = auxValue.match(/[^0-9-]/g).length,
```

Comment:
Why is this needed?

Reply:
The intention was to make wide gauge stat implementation consistent with wide series stat and to prevent value moving with change from positive to negative (-) values. Or do you ask whether it makes sense for a gauge stat since it can't display minus values, e.g. a percentage loss?

Reply:
I wasn't sure what it has to do with `NonDigitCharCount` but now I see it's for the `-` sign 👍